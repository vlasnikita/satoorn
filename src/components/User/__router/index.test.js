import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";

import UserRouter from "./index";

it('should render UserRouter correctly', () => {
    const renderer = new ShallowRenderer();
    const wrapper = renderer.render(<UserRouter match={{params: {username: 'some Login'}}} />)
    expect(wrapper).toMatchSnapshot()
})