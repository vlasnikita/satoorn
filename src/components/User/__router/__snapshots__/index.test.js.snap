// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render UserRouter correctly 1`] = `
<div
  className="User__body"
>
  <span
    className="User__bodyBlur"
  />
  <Route
    component={[Function]}
    path="/some Login/transactions"
  />
  <Route
    component={[Function]}
    path="/some Login/challenges/:challengeId"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/some Login/challenges"
  />
  <Route
    component={[Function]}
    path="/some Login/widgets"
  />
  <Route
    component={[Function]}
    path="/some Login/create"
  />
  <Route
    component={[Function]}
    path="/some Login/channel"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/some Login"
  />
</div>
`;
